### PR TITLE
Update Weibo.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/Weibo.com.xml [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
+		.weibo.com
+		game.weibo.com
+		sg2.game.weibo.com
+		yahoo.tw.weibo.com
 
 	MCB:
-		chart.weibo.com
-		e.weibo.com
+		chart.weibo.com		( Too many redirects )
+		e.weibo.com			( https://e.weibo.com/register/register )
```
Comment (J0WI) [inline src/chrome/content/rules/Weibo.com.xml]: I don't understand why you removed this target

Comment (ivysrono) [inline src/chrome/content/rules/Weibo.com.xml]: https://e.weibo.com/register/register with http://weibo.com/signup/signup.php

Comment (J0WI) [inline src/chrome/content/rules/Weibo.com.xml]: Yes, but it was already in the `mixedcontent` ruleset before. So why did you remove it from there?

Comment (ivysrono) [inline src/chrome/content/rules/Weibo.com.xml]: mixedcontent pages' contents should be https all.
weibo.com/signup/ redirects to http, no https

Comment (J0WI) [inline src/chrome/content/rules/Weibo.com.xml]: Ok, thanks for clarify.
